lodash dependency not imported correctly

On the project, lodash is not a direct dependency. There are different version from the project dependencies (if you try `npm list lodash`), the `2.4.1` (the one we want, from https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/package.json) or the `4.17.21` (not working on the project, used by other dependencies).

NPM will install one of the versions (and put it in the `node_modules/lodash` directory), and the other will be put in `node_modules/<dependency-using-other-lodash-version>/node_modules/lodash`. If the chosen version is not `2.4.1`, then you should have an error when running `npm run test`:
```
Module not found: Error: Can't resolve '[...]/esn-frontend-inbox/node_modules/lodash/dist/lodash.js' in '[...]/esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/models'
```

A simple solution would be to use lodash from `node_modules/<dependency-using-other-lodash-version>/node_modules/lodash` instead of from `node_modules/lodash` but it would not work when the version is the good one.

Or we could import lodash directly.

We also have to check if the problem happens on other repositories.

Quick local workaround if you stumble on the problem : `npm i --no-save lodash@2.4.1`

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
     "esn-frontend-inbox-calendar": "github:openpaas-suite/esn-frontend-inbox-calendar#main",
     "esn-frontend-inbox-linshare": "github:openpaas-suite/esn-frontend-inbox-linshare#main",
     "jmap-draft-client": "^0.1.5",
+    "lodash": "2.4.1",
     "sanitize-html": "github:linagora/sanitize-html#c89d4ebe09da48296f33cc80ddcc02e11d853265",
     "ui-select": "0.19.8",
     "angular-ui-tree": "~2.22.6"

